src/main/java/usecases/RunCourseUseCase/RunCourseInteractor.java

```diff
@@ -0,0 +1,50 @@
+package usecases.RunCourseUseCase;
+
+import entities.Course;
+import entities.CourseFactory;
+import entities.Question;
+import java.util.List;
+import java.util.ArrayList;
+
+public class RunCourseInteractor {
+    private CourseFactory courseFactory;
+    private RunCourseSessionDataAccess sessionDbGateway;
+    private RunCourseOutputBoundary presenter;
+
+    public RunCourseInteractor(CourseFactory courseFactory, RunCourseSessionDataAccess sessionDbGateway, RunCourseOutputBoundary presenter){
+        this.courseFactory = courseFactory;
+        this.sessionDbGateway = sessionDbGateway;
+        this.presenter = presenter;
+    }
+    public boolean answersValid(List<String> answers) {
+        for (int i = 0; i < answers.size(); i++) {
+            if (answers.get(i).equals("")) {
+                return false;
+            }
+        }
+        return true;
```

Comment:
Good! Adding this as a method is good because it is a small method. Originally I know that we planned for ValidAnswers to be a class that would check if the answers are valid. But I like this implementation better because otherwise the other class would be too small.